### ISSUE TITLE
Compatibility with ember-cli-meta-tags

### ISSUE THREAD

**Issue by joeygilkie** [adopted-ember-addons/ember-page-title#124]
This add-on seems to be incompatible with ember-cli-meta-tags. Just tested with a simplistic test case where the title tag is rendered twice, with the lower one being the only one to change.

**Comment by tim-evans** [comment]
Hi there, [this](https://github.com/ronco/ember-cli-meta-tags#using-with-other-ember-cli-head-addons) should help with your issue :)

**Comment by joeygilkie** [comment]
I have that + <title>{{model.title}}</title> in my template/head.hbs file. Still does not work for some reason. There is two titles in my document head. One is before all the style sheets, and one is after. The one that is before does not get updated unless the page is refreshed.

**Comment by tim-evans** [comment]
I don't think I'll be able to debug this with the information you provided. Could you provide some more detail?

**Comment by joeygilkie** [comment]
Sure, so during a project we were trying to add in page titles along with meta tags. So we found it was bugged and decided to make a simple use case to make sure it wasn't something wrong on our end. I made a simple project with a home and about page with links between them. Both pages had a title and the about page had meta tags. I added both required elements to the head.hbs file and was still running into the same problem where there were essentially two ember-head sections being created. The one that is the highest (so the one that actually will be read as the page title) will not change unless the entire page is refreshed. Honestly this could be a bug with ember-head but I simply created this issue to see if there were any known solutions.
